Dockerfile

```diff
@@ -1,11 +1,19 @@
 ## compile proxy-init utility
-FROM golang:1.12.9 as golang
+FROM --platform=$BUILDPLATFORM golang:1.12.9 as golang
```

Comment:
Is `$BUILDPLATFORM` necessary?

Reply:
Yes, because `By default, the target platform of the build request is used.`
Ref: https://docs.docker.com/engine/reference/builder/#from